app/scripts/events.js

```diff
@@ -891,20 +899,23 @@ define([
                 }
             });
 
+            $workspace.on('submit', '.l-message', function(event) {
+                event.preventDefault();
```

Comment:
Also you need add `event.stopPropagation()`
Better use `return false`. It's includes `preventDefault` and `stopPropagation`